.github/workflows/ci_cron_daily.yml

```diff
@@ -35,7 +35,7 @@ permissions:
 jobs:
   tests:
     runs-on: ${{ matrix.os }}
-    if: (github.repository == 'astropy/astropy' && (github.event_name == 'schedule' || github.event_name == 'push' || github.event_name == 'workflow_dispatch' || contains(github.event.pull_request.labels.*.name, 'Extra CI')))
+    if: (github.repository == 'astropy/astropy' && (github.event_name == 'schedule' || github.event_name == 'push' || github.event_name == 'workflow_dispatch' || (github.event_name == 'pull_request' && github.event_name != 'labeled' && contains(github.event.pull_request.labels.*.name, 'Extra CI')) || (github.event_name == 'pull_request' && github.event_name == 'labeled' && github.event.label.name == 'Extra CI')))
```

Comment:
I guess `github.event_name == 'pull_request' && github.event_name != 'labeled'` is double. Or can an event name be equal to two things?

Reply:
Oh ooops

Reply:
I was not careful enough. I guess if it is a label event, it has to be a pull request, since you cannot label a scheduled job or anything. 🤔